### ISSUE TITLE
ES6JSX branch gives no-multi-str errors on JSX

### ISSUE THREAD

**Issue by jergason** [eslint/eslint#1533]
I'm using eslint v0.11.0-alpha.0 and getting `no-mult-str` errors in my JSX template code:

```JavaScript
var React = require('react');

var App = React.createClass({
  render: function() {
    return (
      <div>
        <h1>wat</h1>
      </div>
    );
  }
});

React.render(<App />, document.body);
```

This produces the following errors:

```bash
ಠ_ಠ  ~/code/happytrack λ eslint durp.jsx

durp.jsx
  3:4   error  App is defined but never used                                 no-unused-vars
  6:11  error  Multiline support is limited to browsers supporting ES5 only  no-multi-str
  7:20  error  Multiline support is limited to browsers supporting ES5 only  no-multi-str

✖ 3 problems
```

The first error looks like it is from https://github.com/estools/escope/issues/33, but the last two errors seem incorrect. I expected the es6jsx branch to correctly parse JSX without giving no-multi-str errors. Am I doing something wrong?
